CMakelists.txt

```
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.0)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```

Suggest raising minimum to 3.15 to avoid deprecation warning in CMake.

```bash
$ cmake -DCMAKE_PREFIX_PATH=$HOME/code/c-bootstrap/install/mongo-c-driver-1.25.0 -S. -Bcmake-build
```

Results in this warning (using cmake 3.27.2):
```
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
```

C driver 1.25.0 also specifies 3.15 as the CMake minimum.